python/BUSINESS_SUPPORT_OR_ABOVE_ENABLED/business-support-enabled_test.py

```diff
@@ -0,0 +1,163 @@
+import sys
```

Comment:
Change name to: BUSINESS_SUPPORT_OR_ABOVE_ENABLED_test.py

Reply:
changed name of the file